src/main/java/org/jabref/gui/frame/MainToolBar.java

```diff
@@ -118,12 +123,12 @@ private void createToolBar() {
                 globalSearchBar,
 
                 rightSpacer,
-
+            
                 new HBox(
                         factory.createIconButton(StandardActions.NEW_ARTICLE, new NewEntryAction(frame::getCurrentLibraryTab, StandardEntryType.Article, dialogService, preferences, stateManager)),
                         factory.createIconButton(StandardActions.NEW_ENTRY, new NewEntryAction(frame::getCurrentLibraryTab, dialogService, preferences, stateManager)),
                         createNewEntryFromIdButton(),
-                        factory.createIconButton(StandardActions.NEW_ENTRY_FROM_PLAIN_TEXT, new PlainCitationParserAction(dialogService)),
+                        newEntryFromPlainTextButton, // Use the modified button here
```

Comment:
No need for the comment - it is self explainiing

```suggestion
                        newEntryFromPlainTextButton,
```

(Did you use ChatGPT? This comment is AI generated, isn't it?)